bag_of_words/code/bag_of_words.py

```diff
@@ -0,0 +1,36 @@
+# Read papers and convert them to bag-of-words representations.
+
+indir = "../data/raw/pubmed-txts-mf/"
+outdir = "../data/processed/"
+from sklearn.feature_extraction.text import CountVectorizer
+vectorizer = CountVectorizer()
+import os
+import numpy as np
+import pickle
+
+# Read papers into a list

```

Comment:
I recommend writing this section as two functions: `read_corpus` and `tokenize_corpus`,  and then using an option arg to `tokenize_corpus` to write the corpus to either `csv` or `pkl` formats (or none at all).

Reply:
solved?